src/plugins/vis_types/timeseries/public/application/components/vis_types/timeseries/series.js

```diff
@@ -68,7 +72,7 @@ const TimeseriesSeriesUI = injectI18n(function (props) {
             uiRestrictions={uiRestrictions}
             dragHandleProps={props.dragHandleProps}
           />
-          <div className="tvbAggRow tvbAggRow--split">
+          <div css={tsvbEditorRowStyle(euiTheme)} style={{ paddingLeft: euiTheme.size.xl }}>
```

Comment:
This same `paddingLeft: euiTheme.size.xl` style is used the same way five different times in this PR. Can likely be consolidated in the common.styles.ts file.

Reply:
done